docs/content.zh/docs/dev/table/sourcesSinks.md

```diff
@@ -334,8 +334,8 @@ a reference implementation.
 In particular, it shows how to
 - create factories that parse and validate options,
 - implement table connectors,
-- implement and discover custom formats,
-- and use provided utilities such as data structure converters and the `FactoryUtil`.
+- implement and discover custom formats, and
+- use provided utilities such as data structure converters and the `FactoryUtil`.
```

Comment:
I think the position of word `and` doesn't matter, is there any guidance that the `word` cannot appear in the first position?

Reply:
Sounds good @leonardBang.  I will close the PR.  Thanks.